Pod/Classes/PINProgressiveImage.m

```diff
@@ -350,59 +301,135 @@ - (CFTimeInterval)estimatedRemainingTime
 }
 
 //Must be called within lock
+//Heavily cribbed from https://developer.apple.com/library/ios/samplecode/UIImageEffects/Listings/UIImageEffects_UIImageEffects_m.html#//apple_ref/doc/uid/DTS40013396-UIImageEffects_UIImageEffects_m-DontLinkElementID_9
 - (UIImage *)postProcessImage:(UIImage *)inputImage withProgress:(float)progress
 {
+    UIImage *outputUIImage = nil;
     CGImageRef inputImageRef = CGImageRetain(inputImage.CGImage);
     if (inputImageRef == nil) {
         return nil;
     }
     
-    CIFilter *gaussianFilter = [CIFilter filterWithName:@"CIGaussianBlur"];
-    [gaussianFilter setDefaults];
-    
-    UIImage *outputUIImage = nil;
-    
-    CIContext *processingContext = GPUProcessingContext;
-    if (self.inBackground) {
-        processingContext = CPUProcessingContext;
-    }
-    
-    CGSize maxInput = processingContext.inputImageMaximumSize;
-    CGSize maxOutput = processingContext.outputImageMaximumSize;
+    CGSize maxSize = CGSizeMake(1024, 1024);
```

Comment:
Is this a reasonable max size?  Perhaps so given that the blurring is supposed to be applied only to incompletely scanned progressive images - but what about a fullscreen iPad Pro at a near-final scan?  Perhaps arguably we shouldn't be blurring those scans at all, but I'm wondering if this value can be computed more abstractly.